src/Util/PostgresHelper.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This file is part of the prooph/pdo-event-store.
+ * (c) 2016-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Pdo\Util;
+
+/**
+ * PostgreSQL helper to work with fully qualified table name.
+ */
+trait PostgresHelper
+{
+    /**
+     * @param string $ident
+     * @return string
+     */
+    private function quoteIdent(string $ident): string
+    {
+        $pos = strpos($ident, '.');
+
+        if (false === $pos) {
+            return '"' . $ident . '"';
+        }
+
+        $schema = substr($ident, 0, $pos);
+        $table = substr($ident, $pos + 1);
+
+        return '"' . $schema . '"."' . $table . '"';
+    }
+
+    /**
+     * Splits fully qualified table name by first dot.
+     * @param string $ident
+     * @return string[]
+     */
+    private function splitIdent(string $ident): array
```

Comment:
On reviewing I found that the returned table name is not used at all. So why not change the signature then? Something like `private function schema(string $ident): ?string`. It either returns the schema or null

Reply:
There was `private function extractSchema(string $ident): ?string` in the very first version.
I'll fix this